HomeWork_2/.gitignore

```diff
@@ -0,0 +1,33 @@
+HELP.md
```

Comment:
Please try to use one **.gitignore** file per project (in the root of the project). In case of nested folders please try to use two adjacent asterisk '**'. In case of any questions please contact me in Teams